examples/asr/conf/speech_multitask/fast-conformer_aed.yaml

```diff
@@ -0,0 +1,280 @@
+# It contains the default values for training an autoregressive FastConformer-Transformer ST model with sub-word encoding.
+
+# Architecture and training config:
+# Default learning parameters in this config are set for effective batch size of 2K. To train it with smaller effective
+# batch sizes, you may need to re-tune the learning parameters or use higher accumulate_grad_batches.
+# Here are the recommended configs for different variants of FastConformer-Transformer, other parameters are the same as in this config file.
+# One extra (linear projection) layer is added between FastConformer encoder and Transformer decoder if they have different hidden sizes
+# It is recommended to initialize FastConformer with ASR pre-trained encoder for better accuracy and faster convergence
+
+name: "FastConformer-Transformer-MultiTask"
+
+# Initialize model encoder with pre-trained ASR FastConformer encoder for faster convergence and improved accuracy
+init_from_nemo_model:
+  model0:
+    path: ???
+    include: ["preprocessor", "encoder"]
+
+model:
+  _target_: nemo.collections.asr.models.EncDecMultiTaskModel
+  sample_rate: 16000
+  label_smoothing: 0.0
+  context_len_for_AR_decoding: 5
+  log_prediction: true # enables logging sample predictions in the output during training
+
+  # Important ! Set the prompt format to the class you need
+  prompt_format: ???   # Options supported: ["canary"]
+
+  model_defaults:
+    asr_enc_hidden: 1024
```

Comment:
why `asr` in the name?

Reply:
CC @krishnacpuvvada @stevehuang52

Reply:
I think it's used to distinguish the hidden dim of FC encoder from `lm_enc_hidden` and `dec_hidden`, maybe it's better to use `lm_dec_hidden` to indicate that the latter to belong to the same module? what do you think? @krishnacpuvvada @pzelasko

Reply:
good point! The thought was that, if we introduce additional encoders later, (may be a clap encoder for audio or titanet for speakers)  we would have room to maneuver in-terms of naming later. 
agree with @stevehuang52  wrt to renaming `dec_hidden` to 'lm_dec_hidden`

Reply:
Need to update the checkpoint converters too

Reply:
done, @titu1994 can you update the converter?